integration_test/workflow_tests/test_packages/catalog/test_provider_does_not_exist.py

```diff
@@ -0,0 +1,39 @@
+import json
+import time
+import uuid
+from unittest import TestCase
+
+from requests import Session
+
+import helpers
+
+
+class TestProviderDoesNotExist(TestCase):
+    def test_provider_does_not_exist_returns_empty_granules_list(self):
+        """
+        Presently, we do not raise an error if the providerId is not present in our catalog.
+        """
+        # Set up Orca API resources
+        # ---
+        my_session = Session()
```

Comment:
I wonder if we could take lines 18-24 and add as a helper function to set the session up for you so you would only need to call it like

`my_session = get_request_session()`

Reply:
Added helper